source/wrapper/migrations/1_initial_migration.js

```diff
@@ -1,5 +1,7 @@
-const Migrations = artifacts.require('Migrations')
+const Wrapper = artifacts.require('Wrapper')
 
-module.exports = deployer => {
-  deployer.deploy(Migrations)
+module.exports = (deployer, network) => {
+  if (network == 'rinkeby') {
+    deployer.deploy(Wrapper, '0x43f18D371f388ABE40b9dDaac44D1C9c9185a078', '0xc778417e063141139fce010982780140aa0cd5ab')
```

Comment:
needs to be updated.

const swapAddress = ''
const wethAddress = ""

Reply:
You're looking at an old version of the PR - I think you need to refresh :) This was updated back in 3cbf61b.